omod/src/main/webapp/resources/htmlforms/071-IntegratedAntenatalRegister.xml

```diff
@@ -953,7 +953,7 @@
                             <label>Mebendazole Dose</label>
                             <obs conceptId="9d6abbc4-707a-4ec7-a32a-4090b1c3af87"
                                  answerConceptIds="a7a9d632-b266-4085-9a5e-57fc8dd56f0c,0134c1d3-a255-46b2-ac77-38c0edcd9e53,	beb5523f-ec5a-46fe-a7c9-f3270a05f4b4,dca06bae-30ab-102d-86b0-7a5022ba4115"
-                                 answerLabels="✔ - Received,X - Not Received,ND - Not due,C - Completed"
+                                 answerLabels="&#10003; - Received,X - Not Received,ND - Not due,C - Completed"
```

Comment:
@moshonk This entity also needs to be a tick & #8730; (added a space so that the code can show)